src/components/pager/pager.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import Immutable from 'immutable';
+import classicTheme from '../../style/themes/classic';
+import smallTheme from '../../style/themes/small';
+import Pager from './pager.component';
```

Comment:
you should import `jest-styled-components` to avoid random class name generation in snapshots